src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -701,6 +701,7 @@ cfg_if! {
 pub const RAND_MAX: ::c_int = 0x7fff_ffff;
 pub const PTHREAD_STACK_MIN: ::size_t = 16384;
 pub const SIGSTKSZ: ::size_t = 40960;
+pub const MAP_VPAGETABLE: ::c_int = 0x2000;
```

Comment:
Looking at this comment (https://github.com/DragonFlyBSD/DragonFlyBSD/blob/b0035a16d87ff8c275368a7df9ca8a35cad0a19b/sys/sys/mman.h#L97-L98), I think we don't have to add it on DragonFlyBSD.